three.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
```

Comment:
Very nice job overall! Just a few things:
1. You need more comments, especially before each function saying what it does/return.
2. your code still printed the triangle evaluation (like scalene) for invalid triangles. you shouldn't have had it do that
3. Not something necessarily wrong with your code, but I personally wouldn't have treated the floats as ints